include/unifex/receiver_concepts.hpp

```diff
@@ -116,14 +117,25 @@ namespace _rec_cpo {
           _set_error_fn{}, (Receiver &&) r, (Error&&) error);
     }
     template(typename Receiver, typename Error)
-      (requires (!tag_invocable<_set_error_fn, Receiver, Error>))
+      (requires (!tag_invocable<_set_error_fn, Receiver, Error>) &&
```

Comment:
The `AND` macro lets us do short-circuiting in the concepts emulation layer.

```suggestion
      (requires (!tag_invocable<_set_error_fn, Receiver, Error>) AND
```